apps/platform/src/store/index.ts

```diff
@@ -75,6 +75,8 @@ export const rolesOfWorkspaceAtom = atom<WorkspaceRole[]>([])
 export const selectedApiKeyAtom = atom<ApiKey | null>(null)
 export const apiKeysOfProjectAtom = atom<ApiKey[]>([])
 
+export const projectPrivateKey = atom<string | null>(null)
```

Comment:
rename this to `selectedProjectPrivateKeyAtom`, and remove the extraline on line 77. we want this to be coupled with project atoms